src/PlatformInterface.php

```diff
@@ -15,6 +15,11 @@ interface PlatformInterface extends CommandQuestionInterface {
 
   public const PLATFORM_ALIAS_KEY = 'platform.alias';
 
+  public const GROUP_CONFIG_LOCATION = [
+    '.commonconsole',
+    'groups',
+  ];
```

Comment:
Why do we have to specify the same set of constants in two separate interfaces? Can we just make a common one that extends the two? Or any other way? Is it possible one of them will change? In that case it still can be overwritten if they are inherited.

Reply:
Done.

Reply:
Actually, we are using that constants inside the trait only, so created the array in that particular method only.